### PR TITLE
How to add tutorial to Tutorials page

### PR DESCRIPTION
**Pull request by karengermond**

### CODE REVIEW

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 1]
```diff
@@ -96,6 +96,49 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Where to locate your tutorial
+
+Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature. If you want higher visibility for your tutorial, you can add it to the main Grafana [Tutorials](/tutorials) page. Even though you're displaying your tutorial on the Tutorials page, you need to add your tutorial source to your team's repo to make it easy for team members to review the content.
```
Comment (jdbaldry) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:101]: nit: Splitting sentences onto new lines makes future line based diffing easier to read:
```suggestion
Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature.
If you want higher visibility for your tutorial, you can add it to the main Grafana [Tutorials](/tutorials) page. 
Even though you're displaying your tutorial on the Tutorials page, you need to add your tutorial source to your team's repo to make it easy for team members to review the content. 
```

Comment (jdbaldry) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:101]: nit: I believe this is going to to be the only recommended path going forwards but we can update this once the tutorials have been moved from the grafana/tutorials repository

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 2]
```diff
@@ -96,6 +96,49 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To publish your tutorial to the main Tutorials page:
+
+1. In your documentation repo, create a folder under `sources` called `tutorials` and add your tutorial to this folder.
+
+2. To hide your tutorial from your documentation table of contents, add the following to the metadata of your `sources/_index.md` file (replace your repo for `grafana-cloud`):
+
+    ```
+    cascade:
```
Comment (jdbaldry) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:114]: For this and all the subsequent instructions that refer to YAML, we actually want the user to integrate the snippet with their existing content.

If they already have a `cascade` key, they should not introduce another. Similarly we do not want to introduce a new `manual_mounts` object to the website `config.yaml` but just add a new object to the existing sequence.

In the past I've written something like:

Add the following to the `$.cascade` field in your YAML metadata, creating the `cascade` field if it does not exist:

```yaml
- _target:
    path: /docs/grafana-cloud/tutorials/**
  _build:
    list: false
    render: false
- _target:
    path: /docs/grafana-cloud/**
```

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 3]
```diff
@@ -96,6 +96,54 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+
+Follow the steps below to hide the tutorial from your documentation's table of contents and display it on the Tutorials page.
+
+> Note: This procedure is for writers who have permissions to update the Grafana website repo.
+
+To publish your tutorial to the main Tutorials page:
+
+1. In your documentation repo, create a folder under `sources` called `tutorials` and add your tutorial to this folder.
+
+1. To hide your tutorial from your documentation table of contents, integrate the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file.
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:111]: structurally, this seems like a different task. I'd consider putting this in its own heading.

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 4]
```diff
@@ -96,6 +96,54 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+      _build:
+        list: false
+        render: false
+    - _target:
+        path: /docs/grafana-cloud/**
+    ```
+
+    > Create the `cascade` field if it does not exist. Substitute your repo for `grafana-cloud` in this example.
+
+1. To add your tutorial to the Tutorials page, integrate the following to the `$.manual_mounts` field in the `config/_default/config.yaml` file in the website repo:
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:126]: aren't we already in a task that describes how to add a tutorial to the tutorials page? I think maybe moving step two out into it's own section could help?

**Review by chri2547 — COMMENTED** [review]
Some questions for ya.

File: docs/sources/writing-guide/topic-types/tutorial/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/writing-guide/topic-types/tutorial/index.md]: Specifically this is about where the tutorials live. I read:

> Sometimes it makes sense to publish your tutorial directly within the documentation for your product or feature

and assumed it was referring to alternative place you can publish tutorials -- https://github.com/grafana/tutorials. That alternative is deprecated and all new tutorials should go into their project repositories.

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 5]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Publish your tutorial
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page. 
+
+### Hide your tutorial from your Table of Contents
+
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:106]: I would add an intro here that explains why I would want to hide the tutorial. Do we always want it hidden from the primary docs toc?

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 6]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+Before completing these steps, you must create a `tutorials` directory under `sources` and add your tutorial into its own subdirectory as described in the [Write a tutorial topic](#write-a-tutorial-topic) section.
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:107]: I would put this in a #### Before you begin section.

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 7]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
 
 If getting access to the tutorial data is complex, include the instructions in the steps of the tutorial. If getting access to the data is straightforward, include it in the "Before you begin" section.
 
+## Publish your tutorial
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page.
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:103]:
```suggestion
The following sections describe how to hide the tutorial from your project's table of contents and display it on the Tutorials page. 
```

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 8]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+
+Your tutorial source is stored in your project repo in a `tutorials` folder and mounted to the tutorials repo so that it can be displayed on the [Tutorials](/tutorials) page. The source is stored in your project repo to make it easy for team members to review and edit the content. 
+
+Follow the steps below to hide the tutorial from your project's table of contents and to display it on the Tutorials page. 
+
+### Hide your tutorial from your Table of Contents
+
+Before completing these steps, you must create a `tutorials` directory under `sources` and add your tutorial into its own subdirectory as described in the [Write a tutorial topic](#write-a-tutorial-topic) section.
+
+- Hide your tutorial from your documentation table of contents by integrating the following to the `$.cascade` field in the YAML metadata of your `sources/_index.md` file.
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:109]: You could also make this a multi-step task like so:

1. Open `sources/_index.md`.
1. Add the following code to the `$.cascade` field.

[code here...]

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 9]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+    - _target:
+        path: /docs/grafana-cloud/tutorials/**
+      _build:
+        list: false
+        render: false
+    - _target:
+        path: /docs/grafana-cloud/**
+    ```
+
+    > Create the `cascade` field if it does not exist. Substitute your repo for `grafana-cloud` in this example.
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:122]:
```suggestion
> **Note:** Create the `cascade` field if it does not exist. Substitute your repo for `grafana-cloud` in this example.
```

File: docs/sources/writing-guide/topic-types/tutorial/index.md [hunk 10]
```diff
@@ -96,6 +96,58 @@ For example, the [Play with Grafana Mimir](/tutorials/play-with-grafana-mimir/)
...
+
+    ```yaml
+    manual_mounts:
+      - source: content/docs/grafana-cloud/tutorials/k8s-monitoring-app
+        target: content/tutorials/k8s-monitoring-app
+    ```
+
+    > Create the `manual_mounts` field if it does not exist. Substitute the source and target with your tutorial's path and name.
+
+1. Specify the level and type of your tutorial (substitute the target path of your tutorial):
```
Comment (chri2547) [inline docs/sources/writing-guide/topic-types/tutorial/index.md:140]: Maybe divide this into 2 steps? Like:

1. Specify the tutorial's `level` and `type`.
1. Add the target path to the tutorial content in the `page` attribute.

**Pull request merged**